### PR TITLE
Tests: update screenshots

### PR DESCRIPTION
**Pull request by munrocket**

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
So the random changed behaviour?

File: test/e2e/puppeteer.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ const exceptionList = [
 	'webgl_loader_texture_pvrtc', // not supported in CI, useless
 	'webgl_materials_envmaps_parallax',
 	'webgl_test_memory2', // gives fatal error in puppeteer
-	'webgl_worker_offscreencanvas' // in a worker, not robust
+	'webgl_worker_offscreencanvas', // in a worker, not robust
```
Comment (mrdoob) [inline test/e2e/puppeteer.js:26]: Ohh...

**Comment by munrocket** [comment]
Seems like that, objects placed in another position.

**Pull request merged**